install/collect_mac_addresses_for_ncns.md

```diff
@@ -130,6 +130,8 @@ making a backup of them, in case they need to be examined at a later time.
 
 1. Check that IP addresses are set for each interface and investigate any failures.
 
+  > Note that bond0.can0 is optional in CSM 1.2+
```

Comment:
We should have the same note down around line 317.

Reply:
Fixed